### ISSUE TITLE
supposed way to restart forever

### ISSUE THREAD

**Issue by x-yuri** [foreverjs/forever#684]
I'd appreciate it if I could just:

```
forever restart SCRIPT ARGS
```

But surprisingly it doesn't start the script if it's not already running, so I've got to:

```
forever stop SCRIPT   # here I don't see any sensible option but to ignore all the errors
forever start SCRIPT ARGS
```

The last solution is wordy and possibly forces users to ignore errors they shouldn't ignore. Am I doing it wrong?

**Comment by indexzero** [comment]
No. This is how it works today. This is an interesting feature request that a PR would be welcomed for.
